### ISSUE TITLE
Small nuisance with /run/vdev

### ISSUE THREAD

**Issue by onimatrix81** [jcnelson/vdev#49]
In /etc/vdev/vdevd.conf I left pidfile to be written to /run/vdev/vdev.pid. As good of a file as any afaik. Whose responsibility however is it to create the directory, because vdevd just errors out if it doesn't exist?

**Comment by jaromil** [comment]
this is a good behavior across *nix daemons, creating non existing directories may introduce security rirks

**Comment by fbt** [comment]
Agreed, I have no problems with initscripts creating directories.

**Comment by jcnelson** [comment]
@onimatrix81 The init script should make `/run/vdev`.  Looks like the included init script does not do this; I just pushed a patch to make it do so.

**Comment by onimatrix81** [comment]
Yeah jcnelson. Heh, I had no idea about that. It was "an issue" when I kept starting vdevd manually. My OpenRC init script does this too as Jaromil abd fbt cleared it out that this was desired behaviour.
